### PR TITLE
docs: (Core) add example behaviour to action sheet

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
part of: #3786

#### Please provide a brief summary of this pull request.
There are added some behaviour examples in action sheet.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-compact/action-sheet-compact-example.component.ts [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
     templateUrl: './action-sheet-compact-example.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class ActionSheetCompactExampleComponent {}
+export class ActionSheetCompactExampleComponent {
+
+    @ViewChild(ActionSheetComponent)
+    actionSheetComponent: ActionSheetComponent;
+
+    constructor(public messageToastService: MessageToastService) {}
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-compact/action-sheet-compact-example.component.ts:14]: maybe it should be `private`?))

File: apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-cosy/action-sheet-cosy-example.component.ts [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
     selector: 'fd-action-sheet-cosy-example',
     templateUrl: './action-sheet-cosy-example.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class ActionSheetCosyExampleComponent {
+
+    @ViewChild(ActionSheetComponent)
+    actionSheetComponent: ActionSheetComponent;
+
+    constructor(public messageToastService: MessageToastService) {}
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-cosy/action-sheet-cosy-example.component.ts:14]: the same question

File: apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-compact/action-sheet-compact-example.component.ts [hunk 3]
```diff
@@ -1,8 +1,27 @@
...
+
+    actionPicked(action: string): void {
+        this.openMessageToast(action);
+        this.actionSheetComponent.close();
+    }
+
+    openMessageToast(action: string): void {
+        const content = action +  ' action performed';
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-compact/action-sheet-compact-example.component.ts:22]: `${action} action performed`

File: apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-cosy/action-sheet-cosy-example.component.ts [hunk 4]
```diff
@@ -1,9 +1,27 @@
...
+
+    actionPicked(action: string): void {
+        this.openMessageToast(action);
+        this.actionSheetComponent.close();
+    }
+
+    openMessageToast(action: string): void {
+        const content = action +  ' action performed';
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-cosy/action-sheet-cosy-example.component.ts:22]: the same

File: apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-mobile/action-sheet-mobile-example.component.ts [hunk 5]
```diff
@@ -1,11 +1,30 @@
...
     templateUrl: './action-sheet-mobile-example.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class ActionSheetMobileExampleComponent {}
+export class ActionSheetMobileExampleComponent {
+
+    @ViewChild(ActionSheetComponent)
+    actionSheetComponent: ActionSheetComponent;
+
+    constructor(public messageToastService: MessageToastService) {}
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-mobile/action-sheet-mobile-example.component.ts:17]: the same

File: apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-mobile/action-sheet-mobile-example.component.ts [hunk 6]
```diff
@@ -1,11 +1,30 @@
...
+
+    actionPicked(action: string): void {
+        this.openMessageToast(action);
+        this.actionSheetComponent.close();
+    }
+
+    openMessageToast(action: string): void {
+        const content = action +  ' action performed';
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-mobile/action-sheet-mobile-example.component.ts:25]: the same

**Review by InnaAtanasova — APPROVED** [review]
Please take a look at @Betrozov comments. I approve the rest of the code.

**Comment by JKMarkowski** [comment]
Thanks @Betrozov for review. Those comments has been addressed

**Review by Betrozov — APPROVED** [review]

**Pull request merged**